setup.py

```diff
@@ -1,7 +1,9 @@
 from setuptools import setup
 
+__version__ = '0.0.1'
```

Comment:
Reminder to us for later: we should bump this version number to 1.0.0 before we do an actual release.